codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieDecoder.java

```diff
@@ -143,7 +144,7 @@ public Cookie decode(String header) {
         return cookieBuilder != null ? cookieBuilder.cookie() : null;
     }
 
-    private static class CookieBuilder {
+    private class CookieBuilder {
```

Comment:
@dvlato is there a reason why this can't be static anymore ?

Reply:
This was part of the change we discussed to log a message when the SameSite attribute value was incorrect ... It´s been rolled back now as we don´t need access to the logger (of course we could have created a different,static, logger for the CookieBuilder class as well ).